### DISCUSSION TITLE
is there any restrictions on downloading videos from missav website?

### DISCUSSION THREAD

**Discussion by bennyant** [aclap-dev/video-downloadhelper#399] [Help and Q&A]
### Your question

It was working fine until recently. 
Now when I try to download its showing the following error,

moz-extension://1a80ebf9-6eae-4bef-b370-e001daf939aa/background/main.js:1
receive@moz-extension://1a80ebf9-6eae-4bef-b370-e001daf939aa/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://1a80ebf9-6eae-4bef-b370-e001daf939aa/background/main.js:1:42626

why?

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-GB
Production build
Built on 2024-03-18
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

Platform Win x86-64
Browser Mozilla Firefox 124.0.2

Found companion app: VdhCoApp 2.0.19

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

There is a certain similarity in the name of the web site you cite & the URLs offered in this discussion: #391.  Perhaps something over there might apply to you.  I recommend you read all of that discussion, then provide more details about your problem here.  It would be best if you post a full URL of a web page on which your problematic video is hosted.  Failing that, provide the type of information that other user did.

**Reply by bennyant** [reply]
**Full information:**

URL [https://missav.com/dm13/en/higr-023](url)


Hit Details⊗
bitrate	
2800000
bulk	
undefined
chunked	
hls
core_media	
{...
}
created	
1713075253858
descrPrefix	
HLS streaming
duration	
4930.258666999912
extension	
mp4
group	
grp-7882899e
headers	
[...
]
id	
hls:87286e3
isPrivate	
true
masterManifest	
https://surrit.com/fc34881c-4e41-4d3b-8b09-99125d10ccee/playlist.m3u8
mediaManifest	
https://surrit.com/fc34881c-4e41-4d3b-8b09-99125d10ccee/1280x720/video.m3u8
opStartDate	
1713075258088
operation	
null
originalId	
network-probe:51b79fad
pageUrl	
https://missav.com/dm13/en/higr-023
referrer	
https://missav.com/dm13/en/higr-023
running	
0
size	
1280x720
status	
inactive
tabId	
2
thumbnailUrl	

https://eightcha.com/higr-023/cover.jpg?class=normal
title	
HIGR-023 Spread Wings / Tsubasa Akimoto - MissAV.com |
topUrl	
https://missav.com/dm13/en/higr-023


**This is the fetched video information. 

I am using beta version of VDH (Version 8.2.2.8a1)**

**When I tried to download, it showed the following,**

Exit code: 1

moz-extension://1a80ebf9-6eae-4bef-b370-e001daf939aa/background/main.js:1
receive@moz-extension://1a80ebf9-6eae-4bef-b370-e001daf939aa/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://1a80ebf9-6eae-4bef-b370-e001daf939aa/background/main.js:1:42624

**Reply by GrampaWildWilly** [reply]
As I suspected, this is another porn site.

I got the same error in VDH that you did.  I also tried to see what I could figure out from the Network Monitor but the page is continuously sending manifests to the browser.  The list grows so fast I can't keep the Network Monitor still long enough to analyze anything.

But using the master manifest URL that VDH found, I was able to launch a download using VDH.  I got a 1280x720 mp4, size 873M, duration 1:22:10.  The download took a little under 6 minutes.  It played fine in VLC.

I don't know why this doesn't work properly.  I don't work for VDH.  I'm just another user.  It seems very strange that when you try to use VDH directly on the web page, it doesn't work.  But if you use the master manifest that VDH itself finds, you can use VDH to download the video without problems.  This is a little trick our fellow user @Mattmjs discovered.  Here's the steps:

Open the master manifest URL in its own Firefox window:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/ddca955e-0e7f-46cd-96bd-532904124400)

That gets you this:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/ec17e07f-4de5-4136-83e9-5cd351defc75)

Just hit the Escape key to dismiss the dialog.  Now open the VDH menu:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/a1a73a09-f496-41e9-aede-6f0346e5a37c)

I downloaded the first variant offered by VDH.
